README.md

```diff
@@ -134,19 +189,187 @@ sudo make install
 * Parse traces (note usage of path returned in --list-traces):
 
   ~~~{.sh}
-  iotrace --trace-parser --io --path "kernel/2019-05-10_15:24:21" --format json
+  iotrace --trace-parser --io --path "kernel/2020-07-02_08:52:51" --format json
```

Comment:
Are we referring to included tar file somewhere? Does example trace contains parsed form of traces?